tools/cli/commands/changelog.js

```diff
@@ -743,6 +743,17 @@ async function promptChangelog( argv, needChangelog, types ) {
 		},
 	} );
 
+	let userFacing = false;
+	// Ask if this is a user facing change if we detect Jetpack.
+	if ( argv.project === 'plugins/jetpack' ) {
+		userFacing = await prompt( {
```

Comment:
Having to check `userFacing.userFacing` below is really kind of weird. Possibly we should do it like this:
```js
const { userFacing } = await enquirer.prompt( {
    type: 'confirm',
    name: 'userFacing',
    message: '???',
    initial: true,
    skip: ! needChangelog.includes( 'plugins/jetpack' ) ,
} );
```
As things currently are (cf. [enquirer/enquirer#340](https://href.li/?https://github.com/enquirer/enquirer/issues/340)) this will result in `userFacing` being automatically true for everything except Jetpack, while for Jetpack it'll ask and assign true or false. Then the check below can be just `if ( userFacing ) {`.

Or if you'd rather only call `prompt` for Jetpack instead of using `skip`, we could still do like
```js
( { userFacing } = await prompt( {
    // etc
} ) );
```
or
```js
userFacing = await prompt( {
    // etc
} ).userFacing;
```

Reply:
> Doesn't solve this problem though :/ Not sure what knowledge I'm missing about destructuring objects, or maybe something with how enquirer returns.

Is this it?
* When you do `const { foo } = something;`, JS sees it as destructuring.
* When you do `{ foo } = something;`, JS thinks that's a [block statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block) rather than a destructuring assignment, so it complains about not expecting the `=`.
* The extra parens in `( { foo } = something );` make it see it as destructuring again.